src/main/interface/settings.c

```diff
@@ -970,51 +970,49 @@ const clivalue_t valueTable[] = {
     { "osd_tim1",                   VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, INT16_MAX }, PG_OSD_CONFIG, offsetof(osdConfig_t, timers[OSD_TIMER_1]) },
     { "osd_tim2",                   VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, INT16_MAX }, PG_OSD_CONFIG, offsetof(osdConfig_t, timers[OSD_TIMER_2]) },
 
-    { "osd_vbat_pos",               VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, OSD_POSCFG_MAX }, PG_OSD_CONFIG, offsetof(osdConfig_t, item_pos[OSD_MAIN_BATT_VOLTAGE]) },
```

Comment:
This change will be very disruptive for a lot of users, because they use CLI `diff` to do backup / restore of their config across updates. I am wondering if it wouldn't make sense to leave the names alone, and define the value in such a way that restoring a pre - OSD - profile config will result in all active elements being enabled in the first profile?

Reply:
Yes, but whole packing is on user. With type ARRAY, it is considerably easier to use CLI `set`

Reply:
In this case I am split on the usefulness of allowing users to manipulate the unpacked OSD configuration. In most cases they will want to use the GUI for it, to make sure fields are properly aligned and don't overlap, both of which are hard / impossible to tell even from unpacked data.
In addition to this, when using packed data, the size of the individual components can be chosen to match the available range, making it possible to limit the input range to valid configurations. If an array is used, neither x nor y nor profile will be able to use the full 8 bit input range, and there won't be a way to prevent invalid settings from being entered.

Reply:
With array, user may setup positions with current configurator, dump them and set visibility bits accordingly. With uint16, it's dump, split using calculator, add bits and convert to int again ...

With configurator support, uint16_t will be fine again ...